web/assets/full-width-container.js

```diff
@@ -1,20 +1,15 @@
 const fullWidthContainer = () => {
   if (window.location.pathname.includes("/search")) {
-    const headerContainer = document.getElementsByClassName(
-      "moj-header__container",
-    )[0];
     const mainContainers = document.querySelectorAll(".govuk-width-container");
 
-    if (headerContainer && mainContainers) {
-      headerContainer.style.maxWidth = "none";
-      headerContainer.className =
-        headerContainer.className +
-        " govuk-!-margin-left-5 govuk-!-margin-right-5";
-      mainContainers.forEach(
-        (container) =>
-          (container.className =
-            "govuk-!-margin-left-5 govuk-!-margin-right-5"),
-      );
+    if (mainContainers) {
```

Comment:
Can this be done in SASS/CSS rather than with JS?

Reply:
I need some logic to only expand the width of the header container when on the search results page, I could apply styling to the search results template which would remove the margins and make it full width, but the header would not - so its easier to then just expand everything in here